c_gen/c_code_gen.py

```diff
@@ -99,7 +99,7 @@ def base_h_gen(out, name):
  ****************************************************************/
 
 """)
-    for name, val in of_g.experimenter_name_to_id.items():
+    for name, val in sorted(of_g.experimenter_name_to_id.items()):
```

Comment:
Could possibly change c_gen/of_g_legacy.py#L325 to
```
experimenter_name_to_id = OrderedDict(
    ("bsn", 0x005c16c7),
    ("nicira", 0x00002320),
    ("openflow", 0x000026e1)
    )
```
to avoid consumer-side sorting

Reply:
(* or construct the map as an `OrderedDict(sorted(dict(...).items())`, possibly)

Reply:
yes, changed to OrderedDict( [ ... ] )